kubernetes/helm_charts/datapipeline_jobs/values.j2

```diff
@@ -335,6 +335,39 @@ user-cache-updater: |
   user.self.signin.key = "Self-Signed-In"
   user.valid.key = "Validated"
 
+user-cache-updater_v2: |
+  include file("/data/flink/conf/base-config.conf")
+  kafka {
+    input.topic = {{ env_name }}.telemetry.audit
+    groupId = {{ env_name }}-user-cache-updater-group
```

Comment:
Please use a different consumer group from the v1 version. While we create a new consumer group, please do not add v2 to the consumer group if this job is going to be used going forward.

Reply:
Done